src/pages/interface/index.js

```diff
@@ -15,41 +13,52 @@ import GridToolbar from 'src/components/Shared/GridToolbar'
 
 // ** API
 import { RequestsContext } from 'src/providers/RequestsContext'
-import { getNewInterface, populateInterface } from 'src/Models/RemittanceSettings/Interface'
```

Comment:
getNewInterface and populateInterface are obsolete
delete the files from Models please